src/@next/Input/InputStyle.ts

```diff
@@ -104,7 +105,7 @@ export const StyledInput = styled.input<InputProps>`
   height: 36px;
 
   &::placeholder {
-    color: ${Neutral.B40};
+    color: ${theme.placeholder};
```

Comment:
@franciscrispin can we directly change the color instead? since we are not using this kind of pattern in the code 🙇

Reply:
@ninariccimarie removed the theme constant!

Reply:
thanks for raising this concern! I'll take this into consideration as a pain point for when we need to update